2.4.2/bullseye/build-time-pip-constraints.txt

```diff
@@ -0,0 +1,624 @@
+APScheduler==3.6.3
+Authlib==1.1.0
+Babel==2.10.3
+ConfigUpdater==3.1.1
+Deprecated==1.2.13
+Flask-AppBuilder==4.1.4
+Flask-Babel==2.0.0
+Flask-Bcrypt==1.0.1
+Flask-Caching==2.0.1
+Flask-JWT-Extended==4.4.4
+Flask-Login==0.6.2
+Flask-SQLAlchemy==2.5.1
+Flask-Session==0.4.0
+Flask-WTF==1.0.1
+Flask==2.2.2
+GitPython==3.1.29
+HeapDict==1.0.1
+JPype1==1.4.0
+JayDeBeApi==1.2.3
+Jinja2==3.1.2
+Mako==1.2.3
+Markdown==3.4.1
+MarkupSafe==2.1.1
+PyGithub==1.56
+PyHive==0.6.5
+PyJWT==2.6.0
+PyNaCl==1.5.0
+PyYAML==6.0
+Pygments==2.13.0
+SQLAlchemy-JSONField==1.0.0
+SQLAlchemy-Utils==0.38.3
+SQLAlchemy==1.4.27
+SecretStorage==3.3.3
+Sphinx==5.3.0
+Unidecode==1.3.6
+WTForms==3.0.1
+Werkzeug==2.2.2
+adal==1.2.7
+aiofiles==0.8.0
+aiohttp==3.8.3
+aiosignal==1.2.0
+alabaster==0.7.12
+alembic==1.8.1
+aliyun-python-sdk-core==2.13.36
+aliyun-python-sdk-kms==2.16.0
+amqp==5.1.1
+analytics-python==1.4.0
+ansiwrap==0.8.4
+anyio==3.6.2
+apache-airflow-providers-airbyte==3.1.0
+apache-airflow-providers-alibaba==2.1.0
+apache-airflow-providers-amazon==6.0.0
+apache-airflow-providers-apache-beam==4.0.0
+apache-airflow-providers-apache-cassandra==3.0.0
+apache-airflow-providers-apache-drill==2.2.1
+apache-airflow-providers-apache-druid==3.2.1
+apache-airflow-providers-apache-hdfs==3.1.0
+apache-airflow-providers-apache-hive==4.0.1
+apache-airflow-providers-apache-kylin==3.0.0
+apache-airflow-providers-apache-livy==3.1.0
+apache-airflow-providers-apache-pig==3.0.0
+apache-airflow-providers-apache-pinot==3.2.1
+apache-airflow-providers-apache-spark==3.0.0
+apache-airflow-providers-apache-sqoop==3.0.0
+apache-airflow-providers-arangodb==2.0.0
+apache-airflow-providers-asana==2.0.1
+apache-airflow-providers-atlassian-jira==1.0.0
+apache-airflow-providers-celery==3.0.0
+apache-airflow-providers-cloudant==3.0.0
+apache-airflow-providers-cncf-kubernetes==4.4.0
```

Comment:
Do we need a repackaged version of this with Istio?

Reply:
I think it's already available hence why the image passed